### ISSUE TITLE
Training Loss

### ISSUE THREAD

**Issue by dragonlzm** [PolyU-ChenLab/ETBench#10]
![Image](https://github.com/user-attachments/assets/4e037dc6-32de-4489-bc14-f2e361db60d4)
Hi, thanks for your great work! I recently tried to reproduce the stage-3 training using the command `bash scripts/train_stage_3.sh PolyU-ChenLab/ETChat-Phi3-Mini-Stage-2`. I noticed that the training loss at the very beginning of the training is about 20-ish, which is much larger than I expected (as in the picture I attached). Just try to confirm whether this is a normal range of the size of the loss. Thanks!

**Comment by yeliudev** [comment]
Hi @dragonlzm! Many thanks for your interest in our work. Yes the high loss is expected, as it is a combination of language modeling loss (for next-token prediction) and embedding matching loss (for temporal grounding). You may compare it with [our training log](https://huggingface.co/PolyU-ChenLab/ETChat-Phi3-Mini-Stage-3/raw/main/trainer_state.json).
